frontend/package.json

```diff
@@ -47,16 +47,17 @@
     "test:unit:devtools": "cross-env VITE_NODE_ENV=test NODE_ENV=test vitest run --inspect-brk --pool threads --poolOptions.threads.singleThread"
   },
   "dependencies": {
-    "@bcgov-nr/nr-fsa-theme": "^1.1.3",
-    "@bcgov/bc-sans": "^2.0.0",
+    "@bcgov-nr/nr-fsa-theme": "^1.2.0",
+    "@bcgov/bc-sans": "^2.1.0",
     "@carbon/icons-vue": "^10.99.1",
     "@carbon/pictograms": "^12.42.0",
     "@carbon/styles": "^1.67.0",
     "@carbon/web-components": "^2.12.0",
     "@vueuse/core": "^12.5.0",
-    "aws-amplify": "^6.6.6",
+    "aws-amplify": "^6.13.3",
     "axios": "^1.7.2",
-    "date-fns": "^4.0.0",
+    "date-fns": "^4.1.0",
+    "dompurify": "^3.2.4",
```

Comment:
DO we use this dependency directly or is it a transient dependency? If it's a transient dependency, this is being handled by another PR and should not be added directly.